www/package.json

```diff
@@ -64,7 +64,7 @@
     "@types/oauth": "^0.9.1",
     "autoprefixer": "^10.4.7",
     "dotenv": "^16.0.1",
-    "next-auth": "~4.10.3",
+    "next-auth": "~4.13.0",
```

Comment:
cc @sachinraja why is it only picking up 1 package.json? :(

Reply:
Renovate's ignoring the examples:
https://docs.renovatebot.com/presets-default/#ignoremodulesandtests

Reply:
We can also remove the ~ and use `^4.13.3`. `next-auth` has removed the exact next@12.2.5 constraint on their peerDep now.